react-app/src/components/Layout/index.tsx

```diff
@@ -119,6 +121,8 @@ export const Layout = () => {
   const isDesktop = useMediaQuery("(min-width: 768px)");
   const { data: user } = useGetUser();
   const customUserRoles = user?.user?.["custom:cms-roles"];
+  const { pathname } = useLocation();
+  useScrollToTop(pathname);
```

Comment:
I'd also add perhaps we can make this into a component called `ScrollToTop` and position it in `main.ts`? This functionality not strictly a layout thing, and we can remove it from all the other places it's being used in e.g. `DetailsSection`, `Dashboard` etc

Reply:
I believe layout is where it has to live, from what I gathered layout will affect all the other places such as Dashboard and Details section so I removed it as well.

Here's a screenshot of trying to include it in main.ts instead:
![Screen Shot 2024-10-30 at 10 07 00 AM](https://github.com/user-attachments/assets/9cc91a6c-9996-478c-89aa-7ba2403d1147)

Reply:
Yup, that makes sense. I wrongly assumed there's a wrapper for the routes